### DISCUSSION TITLE
Radius for ScatterChart

### DISCUSSION THREAD

**Discussion by TimThieffenat** [imaNNeo/fl_chart#1364] [Q&A] [answered]
I want to do a dots with a radius of 1 meter for exemple (my axes are in meter), but if i put 1 for the radius of the scatterSpot, i will changed with the height and width of the chart. I first though of divising the radius by max(height, width), and it works, the radius is constant, but it's not 1 meter, i need a factor (between 100 and 500 i think) to adjust the size.

**Comment by imaNNeo** [comment, marked as answer]
Yes, I see.
Maybe it's better to change our measurement of radius from pixels to axis-base. Also we can allow user to choose between have radius in axis-based measurement system or pixels measurement system

**Reply by imaNNeo** [reply]
Can you please create an issue for that and mention this discussion?
